README.md

```diff
@@ -54,65 +71,88 @@ uri is correct.  If it needs a trailing slash, be sure to include a trailing sla
 
 Edit the new file (installed in the config directory) and set:
 - `CLIENT_ID` to the client id of your developer application
-- `PERSONAL_ACCESS_TOKEN` to the newly generated token (Only required or recognized for the LOCAL backend; do not set this value for staging, production, or test backends)
-- REDIRECT_URI: Must exactly match the redirect URI used to register the OAuth developer application. 
+- `PERSONAL_ACCESS_TOKEN` to the newly generated token (Only required or recognized for the LOCAL backend; do not set 
+this value for staging, production, or test backends)
+- `REDIRECT_URI`: Must exactly match the redirect URI used to register the OAuth developer application. 
 Default value is appropriate for local development using `ember server`, with a login page at `/login` 
 
-Because of the potential for this file to include sensitive information, we strongly recommend adding this file to 
-`.gitignore` for your project.
+Because of the potential for this file to include sensitive information, **we strongly recommend adding this file to 
+`.gitignore`** for your project.
 
-#### Using the Test API
+#### Alternate option: Environment variables
+If you do not wish to use file-based configuration, any of the settings above can be overridden individually as 
+environment variables with the same name as appears in the config file. Eg
 
-To do this, you will need to [create a developer application](https://test.osf.io/settings/applications/) on the relevant version of the OSF.
+`BACKEND=test CLIENT_ID=gibberish ember serve`
```

Comment:
Changed in ce6fc30

It doesn't affect functionality, but I tend to prefer the "official" forms of commands when I write docs. It's a bit easier for beginners if they don't have to look up aliases.